### ISSUE TITLE
Accessibility issue with hidden connection warning

### ISSUE THREAD

**Issue by paulGeoghegan** [zauberzeug/nicegui#3050]
### Description

I have just started using nicegui and I really like it so far. I'm quite impressed with the native mode and the accessibility which a lot of other options lack. There is one weird thing so far however.
I'm blind so I use a screen reader.
When I open the native app this text appears down the bottom:
```
⚠️
Connection lost.Trying to reconnect...
```
I assume this text is meant to be hidden to users but it isn't to screen reader users.

**Comment by falkoschindler** [comment]
Thank you so much for reaching out, @paulGeoghegan! Although we're aware of screen readers and the importance of accessibility, it's easy to overlook when not using them ourselves. So we really appreciate your feedback.

The "Connection lost" popup is meant to appear only when the connection to the server is lost. This is implemented setting the opacity from 0 to 1, which indeed isn't very screen reader friendly. I'm going to look into this and see if I can make it more accessible.

**Comment by paulGeoghegan** [comment]
@falkoschindler  perfect thanks so much.

**Comment by falkoschindler** [comment]
I think I fixed the issue using the `aria-hidden` attribute as well as setting the CSS property "visibility" accordingly. The fading transition as well as the transition delay before showing the popup has been preserved.
